WordPress/src/main/java/org/wordpress/android/util/WPMediaUtils.java

```diff
@@ -440,6 +422,7 @@ public void onScanCompleted(String path, Uri uri) {
     /*
      * returns true if the current user has permission to upload new media to the passed site
      */
+    @SuppressWarnings("SimplifiableIfStatement")
```

Comment:
This doesn't look like a very complicated if statement, why don't we just simplify it?

Reply:
I felt like the comment `self-hosted sites don't have capabilities so always return true` won't be so clear, but you are right I guess.